pkg/cryptoauthlib/Kconfig

```diff
@@ -23,6 +23,11 @@ config MODULE_CRYPTOAUTHLIB_CONTRIB
 config MODULE_CRYPTOAUTHLIB_TEST
     bool "Module for cryptoauthlib tests"
     depends on TEST_KCONFIG
+    select MODULE_CRYPTOAUTHLIB_TEST_THIRD_PARTY_UNITY
+    select MODULE_CRYPTOAUTHLIB_TEST_API_ATCAB
+    select MODULE_CRYPTOAUTHLIB_TEST_API_CALIB
+    select MODULE_CRYPTOAUTHLIB_TEST_API_CRYPTO
+    select MODULE_CRYPTOAUTHLIB_TEST_VECTORS
```

Comment:
Could you quickly explain what this is doing? Where is it utilized?

Reply:
The library contains folders for different tests for all the APIs (eg. test/api/atcab, test/api/calib, etc). 
For each test folder a module needs to be defined.
When building with Kconfig, we need to select all the folders we want to build. Otherwise the functions are not built and linked correctly.